### PR TITLE
docs: Clarify mesos driver, also line-wrapping

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
cc:@rgardler

### CODE REVIEW

**Comment by rgardler** [comment]
Yep, that's the problem I hit :-)

Thanks for adding it to the docs.

File: cluster/mesos/README.md [hunk 1]
```diff
@@ -8,33 +11,48 @@ containers on a Mesos cluster.
...
+        --cluster-opt mesos.port=3375 \
+        zk://<mesos_masters_url>
 ```
 
-For example, if you run the manager locally on your machine:
+Once the manager is running, check your configuration by running `docker info`
+as follows:
 
 ```
+$ docker -H tcp://<manager_ip:manager_port> info
```
Comment (thaJeztah) [inline cluster/mesos/README.md]: just a nit, so only update if there are other comments; I think this should be `<manager_ip>:<manager_port>` to be consistent

File: cluster/mesos/README.md [hunk 2]
```diff
@@ -8,33 +11,48 @@ containers on a Mesos cluster.
...
+swarm manager, for example 2375. And a second TCP port to listen for the Mesos
+master, for example 3375.
 
 ## Start the Docker Swarm manager
 
 If you use a single Mesos master:
 
 ```     
-docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 <mesos_master_ip>:<mesos_master:port>
+docker run -d -p <swarm_port>:2375 -p 3375:3375 \
```
Comment (thaJeztah) [inline cluster/mesos/README.md]: also just a nit (and I see it was already like this); this should probably have a prompt, like we do in the other docs (`$ docker run....`)

File: cluster/mesos/README.md [hunk 3]
```diff
@@ -8,33 +11,48 @@ containers on a Mesos cluster.
...
+such as `localhost` or some IP address/hostname. `<public_machine_ip>` is the
+IP address for Mesos master to talk to swarm manager. If they are co-located on
+the same machine this value can be `0.0.0.0` or `localhost`.
 
-Once the manager is running, check your configuration by running `docker info` as follows:
+If you use multiple Mesos masters:
 
 ```
-docker -H tcp://<manager_ip:manager_port> info
+docker run -d -p <swarm_port>:2375 -p 3375:3375 \
```
Comment (thaJeztah) [inline cluster/mesos/README.md]: Same here (prompt)

**Comment by thaJeztah** [comment]
Thanks @ahmetalpbalkan, LGTM, marked some nits, but only fix those if there are other comments

**Comment by MHBauer** [comment]
Change all instances of <swarm_port> to <manager_port> . They represent the same thing.

**Comment by ahmetalpbalkan** [comment]
@mhbauer I believe your comment needs syntax escaping.

**Comment by ahmetalpbalkan** [comment]
@thaJeztah fixed your comments.

**Comment by thaJeztah** [comment]
Thanks, @ahmetalpbalkan, re-LGTM

**Comment by MHBauer** [comment]
@ahmetalpbalkan fixed my comment. Still learning markdown.

**Comment by ahmetalpbalkan** [comment]
@MHBauer addressed your comment.

**Comment by thaJeztah** [comment]
> (What do angle brackets mean in markdown?)

Markdown allows embedding HTML markup; so `<foobar>` will be treated as a HTML element, thus become invisible :-)

**Comment by vieux** [comment]
LGTM ping @moxiegirl

File: cluster/mesos/README.md [hunk 4]
```diff
@@ -8,33 +11,48 @@ containers on a Mesos cluster.
...
+        --cluster-opt mesos.port=3375 \
+        <mesos_master_ip>:<mesos_master:port>
 ```
 
-If you use multiple Mesos masters:
-
-```
-docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 zk://<mesos_masters_url>
-```
+The command above creates a swarm manager listening at `<swarm_port>` and
```
Comment (moxiegirl) [inline cluster/mesos/README.md]: Deconstruct these compound sentences to make it easier to read:

--->

The command above creates a swarm manager listening at `<swarm_port>`. The `<mesos_master_ip>` value points to where Mesos master lives in the cluster. Typically, this is localhost, a hostname, or an IP address.  The `<public_machine_ip>` value is the IP address for Mesos master to talk to swarm manager. If master and manager are co-located on the same machine, you can use the `0.0.0.0` or `localhost` value.

**Comment by moxiegirl** [comment]
@ahmetalpbalkan nice...one small change for readability.

**Comment by ahmetalpbalkan** [comment]
@moxiegirl incorporated your comments.

**Comment by moxiegirl** [comment]
LGTM @ahmetalpbalkan thank you for doing this...

**Comment by thaJeztah** [comment]
Thanks Ahmet!

**Comment by ahmetalpbalkan** [comment]
@thaJeztah @moxiegirl np. I guess we can still improve mesos docs a lot further as more people go through it.

**Pull request merged**